### PR TITLE
Add ansible configuration for AIX machines

### PR DESCRIPTION
**Pull request by mhdawson**
Add initial ansible configuration for AIX machines and include instructions for manual bootstrap.

### CODE REVIEW

**Comment by mhdawson** [comment]
@jbergstroem ready for your review

**Comment by mhdawson** [comment]
@jbergstroem  ping

**Comment by jbergstroem** [comment]
@mhdawson sorry, just busy with work. Others -- feel free to review as well; otherwise I'll do it as quick as possible.

File: setup/aix61/resources/S20jenkins [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# purpose: script that will start or stop jenkins
+##################################################
+
+case "$1" in
+start )
+        su - {{user}} -c 'ulimit -c 0; \
+              export LIBPATH=/opt/freeware/lib:$LIBPATH; \
+              export PATH=/usr/java71_64/jre/bin:$PATH; \
+              export HOME=/home/{{user}}; \
+              export NODE_COMMON_PIPE="$HOME/test.pipe"; \
```
Comment (jbergstroem) [inline setup/aix61/resources/S20jenkins]: also set `NODE_TEST_DIR` (usually $HOME/tmp)

**Comment by mhdawson** [comment]
@jbergstroem added setup of NODE_TEST_DIR

File: setup/aix61/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# io.js Build AIX61 Setup
+
+For setting up an AIX 61 box
+
+```text
+Host test-ibm-aix61-ppc64-1:18822
+  HostName cloud.siteox.com
+  User root
+  This machine is a temporary one until we get additional resources at osusol. It is some
```
Comment (jbergstroem) [inline setup/aix61/README.md]: Could you move this text out of the code? The pattern seems to be using code for copy/paste into ssh config.

**Comment by jbergstroem** [comment]
not sure how `/etc/security/login.cfg` looks, but could perhaps [lineinfile](http://docs.ansible.com/ansible/lineinfile_module.html) simplify this?

File: setup/aix61/README.md (hunk 2 above)
Comment (mhdawson) [inline setup/aix61/README.md]: Ok, updated, removed text and added port which is required for ssh config

**Comment by mhdawson** [comment]
@jbergstroem in respect to lineinfile I search on bullfreeware and I don't see it ported there so I don't think its available for AIX.

The line in question looks like this:

 shells = /bin/bash,/bin/sh,/bin/bsh,/bin/csh,/bin/ksh,/bin/tsh,/bin/ksh93,/usr/bin/sh,/usr/bin/bsh,/usr/bin/csh,/usr/bin/ksh,/usr/bin/tsh,/usr/bin/ksh93,/usr/bin/rksh,/usr/bin/rksh93,/usr/sbin/uucp/uucico,/usr/sbin/sliplogin,/usr/sbin/snappd


were I need to add /bin/bash since it is not there by default.

**Comment by mhdawson** [comment]
@jbergstroem wondering if you have more comments ?

**Comment by jbergstroem** [comment]
Not sure what init system aix is running but is output from java being picked up? That'd be my only remaining feedback as of now.

**Comment by mhdawson** [comment]
When you say "is output from java being picked up" can you clarify ?  Do you mean do we capture it and send it to a log etc ?  If so I'll look at what we do for other platforms.

**Comment by jbergstroem** [comment]
@mhdawson on other platforms we usually pipe to a logfile; check other examples.

Btw, I just compiled ccache manually; if you could add /usr/lib/ccache preceding $PATH in your init script and possibly add the ccache compilation to that manual step document it'd be great.

**Comment by jbergstroem** [comment]
Come to think of it; I recall the slave having a `-slaveLog` option. On init systems that doesn't redirect stdout/err we should probably implement that instead.

**Comment by mhdawson** [comment]
I had looked at some other platforms and added the redirect to a log file like they had.  Hope that is good for now.  @jbergstroem if thats ok then I'm hoping this is ready to go in

**Comment by jbergstroem** [comment]
Is there a global cert store in aix? It's a bit unfortunate that we can't validate certs in the playbook.

**Comment by jbergstroem** [comment]
Other than that, LGTM.

**Comment by mhdawson** [comment]
I'll look into the cert side, but as discussed unless somebody else objects I'll squahs land this  tomorrow or Friday.

**Comment by mhdawson** [comment]
Landed as f5a5dd95887ab0827a3adf03c9462e80fe619cc0

**Pull request closed without merging**